sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -2038,7 +2038,8 @@ def PatchItem(
         if options is None:
             options = {}
 
-        headers = base.GetHeaders(self, self.default_headers, "patch", path, document_id, resource_type, options)
+        headers = base.GetHeaders(self, self.default_headers, "patch", path, document_id, resource_type,
```

Comment:
just curious - why all small letters (patch) when the OpType is defined with capital letters here. Don't we have to do case-insensitive comaprison due to it everywehre - https://github.com/Azure/azure-sdk-for-python/blob/f2f1e980572517e34cfb4b7bbb01ae0f7bd9a763/sdk/cosmos/azure-cosmos/azure/cosmos/documents.py#L347-L364

Reply:
the verb here (lower-case) is only used for branching on header logic within the GetHeaders method, it doesn't actually get set anywhere for anything:
![image](https://github.com/user-attachments/assets/ad071165-e134-4345-921c-c98f28b477ee)

 the _OperationType meanwhile does get attached to the request object and is also the value we set in the new header we added (thin-client-operation-type)